cylc/flow/config.py

```diff
@@ -2341,3 +2342,25 @@ def process_metadata_urls(self):
                 self.workflow, cfg['meta']['URL'])
             cfg['meta']['URL'] = RE_TASK_NAME_VAR.sub(
                 name, cfg['meta']['URL'])
+
+    @staticmethod
+    def check_for_owner(tasks: Dict) -> None:
+        """Raise exception if [runtime][task][remote]owner
+        """
+        owners = {}
+        for task, tdef in tasks.items():
+            remote = tdef.get('remote', None)
+            if remote:
+                owner = remote.get('owner', None)
+                if owner:
+                    owners[task] = owner
+        if owners:
+            msg = (
+                '"[runtime][task][remote]owner" is deprecated at Cylc 8.'
```

Comment:
Obsolete seems more accurate?
```suggestion
                '"[runtime][task][remote]owner" is obsolete at Cylc 8.'
```